airflow-core/src/airflow/api_fastapi/common/parameters.py

```diff
@@ -379,14 +379,15 @@ def filter_param_factory(
     default_factory: Callable[[], T | None] | None = None,
     skip_none: bool = True,
     transform_callable: Callable[[T | None], Any] | None = None,
+    description: str | None = None,
```

Comment:
```suggestion
    *,
    description: str | None = None,
```

We probably should have made it keyword-only long ago. but not a bad idea to start from new arguments

Reply:
Done! Just added unit test and make `parameter` keyword-only.